spec/requests/api/v2/viewing_one_application_spec.rb

```diff
@@ -0,0 +1,77 @@
+describe 'GET one application' do
+  let(:json_response) { JSON.parse(response.body) }
+
+  before do
+    headers = { "ACCEPT" => "application/json" }
+    get "/api/v2/applications/#{id}", headers: headers
+  end
+
+  context 'with an id matching the first application' do
+    let(:id) { "3fa85f64-5717-4562-b3fc-2c963f66afa6" }
+
+    it 'responds with a success code' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'responds with the right application' do
+      expect(json_response).to include(
```

Comment:
These can be made a bit nicer with RSpec JSON matchers: https://relishapp.com/waterlink/rspec-json-expectations/docs/json-expectations/include-json-matcher

Reply:
Nice - going to address this in a different PR as it affects the v1 specs as well.